pkgs/development/python-modules/cypari/default.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  fetchurl,
+  setuptools,
+  cython,
+  bash,
+  perl,
+  gnum4,
+  texliveBasic,
+}:
+
+let
+  pariVersion = "pari-2.15.4";
+  gmpVersion = "gmp-6.3.0";
+
+  pariSrc = fetchurl {
+    url = "https://pari.math.u-bordeaux.fr/pub/pari/OLD/2.15/${pariVersion}.tar.gz";
```

Comment:
Maybe should change the pariVersion to 2.15.4, and the link should change "https://pari.math.u-bordeaux.fr/pub/pari/OLD/${lib.versions.majorMinor pariVersion}/pari-${pariVersion}.tar.gz";